src/stan/services/pathfinder/multi.hpp

```diff
@@ -77,6 +77,14 @@ namespace pathfinder {
  * @param[in,out] parameter_writer output for parameter values
  * @param[in,out] diagnostic_writer output for diagnostics values,
  * `error_codes::SOFTWARE` for failures
+ * @param[in] return_lp Whether single pathfinder should return lp calculations.
+ *  If `true`, calculates the joint log probability for each sample.
+ *  If `false`, (`num_draws` - `num_elbo_draws`) of the joint log probability
+ *  calculations will be `NA` and psis resampling will not be performed.
+ * @param[in] psis_resampling If `true`, psis resampling is performed over the
+ *  samples returned by all of the individual pathfinders and `num_multi_draws`
+ *  samples are returned. If `false`, no psis resampling is performed
+ *  and (`num_paths` * `num_draws`) samples are returned.
```

Comment:
They're not really returned, but written to the output writer. 

Might be worth noting that this is equivalent to concatenating the end values of of the `single_path_parameter_writer`s if they were specified, with less computation

Reply:
Agree going to update the docs. return is def not the right word